src/view/trip-item.js

```diff
@@ -1,27 +1,40 @@
 import AbstractView from "./abstract.js";
 
+const createOfferTemplate = (type) => {
+  if (type.offers) {
+    return `<span class="event__offer-title">${type.offers[1].title}</span>
+    &plus;&euro;&nbsp;
+    <span class="event__offer-price">${type.offers[1].price}</span>`;
+  } else {
+    return ``;
+  }
```

Comment:
Раз используется ранний `return`, то и `else` лучше не использовать, т.к. ветвления по-сути уже нет (если дошли до этого момент, то первый кейс не отработал), но сложнее читать.
```suggestion
  }
  
  return ``;
```